requirements.txt

```diff
@@ -25,20 +22,16 @@ isodate==0.6.0            # via gremlinpython
 itsdangerous==1.1.0       # via flask
 jinja2==2.10.3            # via flask
 jsonschema==3.2.0         # via flask-restplus
-locust==0.0
-locustio==0.13.2          # via locust
 markupsafe==1.1.1         # via jinja2
 more-itertools==8.0.2     # via zipp
-msgpack-python==0.5.6     # via locustio
-https://github.com/fabric8-analytics/osa-data-model/archive/master.zip
```

Comment:
So now we are not using osa-data-model anywhere ?

Reply:
Yes. We were only using gremlin adapter earlier., that has been removed to have a better control over error handling.